build/set_build_version.ts

```diff
@@ -18,38 +18,65 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
-import * as fs from "fs";
+import * as fse from "fs-extra";
 import * as path from "path";
 import appInfo from "../package.json";
 import semver from "semver";
 import fastGlob from "fast-glob";
 
 const packagePath = path.join(__dirname, "../package.json");
 const versionInfo = semver.parse(appInfo.version);
-const buildNumber = process.env.BUILD_NUMBER || "1";
-let buildChannel = "alpha";
+const buildNumber = process.env.BUILD_NUMBER || Date.now().toString();
 
-if (versionInfo.prerelease) {
-  if (versionInfo.prerelease.includes("alpha")) {
-    buildChannel = "alpha";
-  } else {
-    buildChannel = "beta";
+function getBuildChannel(): string {
+  switch (versionInfo.prerelease[0]) {
+    case "beta":
+      return "beta";
+    case "rc":
```

Comment:
Rc was not included on purpose. We should have just `alpha`, `beta` and `latest` (stable).

Reply:
Okay will remove and add a comment about that.